### PR TITLE
Update source metadata and enable links to paper

### LINKED ISSUE #513: Source Data improvements
I have provide the information in below format:
Authors | Title | DOI | Year

Yoshiharu Muto; Parker C. Wilson; Nicolas Ledru; Haojia Wu; Henrik Dimke; Sushrut S. Waikar; Benjamin D .Humphreys | Single cell transcriptional and chromatin accessibility profiling redefine cellular heterogeneity in the adult human kidney | https://doi.org/10.1038/s41467-021-22368-w |  2021

Zhong Yi Wang; Adrian Keogh; Annick Waldt; Rachel Cuttat; Marilisa Neri; Shanshan Zhu; Sven Schuierer; Alexandra Ruchti; Christophe Crochemore; Judith Knehr; Julie Bastien; Iwona Ksiazek; Daniel Sánchez Taltavull; Hui Ge; Jing Wu; Guglielmo Roma; Stephen B. Helliwell1; Deborah Stroka; Florian Nigsch | Single‑cell and bulk transcriptomics of the liver reveals potential targets of NASH with fibrosis | https://doi.org/10.1038/s41598-021-98806-y | 2021

Felipe A. Vieira Braga; Gozde Kar; Marijn Berg, Orestes A. Carpaij; Krzysztof Polanski; Lukas M. Simon; Sharon Brouwer; Tomas Gomes; Laura Hesse; Jian Jiang | A cellular census of human lungs identifies novel cell states in health and in asthma | https://doi.org/10.1038/s41591-019-0468-5 | 2019

### PR DESCRIPTION
**Pull request by edlu77**
Closes #513

### CODE REVIEW

**Review by LibbyUX — CHANGES_REQUESTED** [review]
This source data section is looking amazing, @edlu77 ! I am loving the updated collapse functionality. I am also loving the logic of seeing one row if there's only one data source. Exquisite!

A couple of hopefully quick CSS requests:
- Initially, when I designed the new source data area, the paper title was supposed to link out elsewhere, but it appears this may have changed due to the data we have available. These paper titles may need to use our [table text button](https://www.figma.com/design/gQEMLugLjweDvbsNNUVffD/Design-System-Repository?node-id=2319-4362&t=ZvRBz2QZBMH9Rdv7-4) component in future versions. For now, can you please use our HRA blue (it appears black) for plain text in the table here instead?
- Question: Is the DOI link button using the 500 weight from our [table text button](https://www.figma.com/design/gQEMLugLjweDvbsNNUVffD/Design-System-Repository?node-id=2319-4362&t=ZvRBz2QZBMH9Rdv7-4)? It appears a little thicker currently.

525EE666-8120-4AC9-ADDA-4F601110D349


image

**Review by LibbyUX — APPROVED** [review]
Beautiful! Thank you @edlu77

File: libs/components/molecules/src/lib/source-list/source-list.component.html [hunk 1]
```diff
@@ -61,22 +61,25 @@
...
-        (click)="sourceLinkClicked(row)"
         [hraHover]="dataHover"
         [hraHoverData]="row.link"
+        (click)="sourceLinkClicked(row)"
       >
-        {{ row.link }}
+        <a
+          class="source-link"
+          mat-flat-button
+          [href]="sourceLink"
```
Comment (axdanbol) [inline libs/components/molecules/src/lib/source-list/source-list.component.html:76]: Replace with `[href]="row.link"`

File: libs/components/molecules/src/lib/source-list/source-list.component.ts [hunk 2]
```diff
@@ -73,7 +75,10 @@ export class SourceListComponent<T extends SourceListItem> implements OnChanges
   dataSource = new MatTableDataSource<T>();
 
   /** Columns to display in the sources list */
-  displayedColumns: string[] = ['select', 'authors', 'year', 'title', 'doi', 'link'];
+  displayedColumns: string[] = ['select', 'authors', 'year', 'title', 'link'];
+
+  /** Link to selected data */
+  sourceLink = '';
```
Comment (axdanbol) [inline libs/components/molecules/src/lib/source-list/source-list.component.ts:81]: Remove

File: libs/components/molecules/src/lib/source-list/source-list.component.ts [hunk 3]
```diff
@@ -112,6 +117,7 @@ export class SourceListComponent<T extends SourceListItem> implements OnChanges
    * @param item Source list item
    */
   sourceLinkClicked(item: SourceListItem): void {
+    this.sourceLink = item.link;
```
Comment (axdanbol) [inline libs/components/molecules/src/lib/source-list/source-list.component.ts:120]: Remove

**Review by axdanbol — CHANGES_REQUESTED** [review]

**Review by axdanbol — APPROVED** [review]

**Pull request merged**